modules/ossm-about-discoveryselectors.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+// install/ossm-installing-openshift-service-mesh.adoc
+
+:_mod-docs-content-type: Concept
+[id="ossm-about-discoveryselectors_{context}"]
+= About DiscoverySelectors
+:context: ossm-about-discoveryselectors
+
+`discoverySelectors` let the mesh administrator control which namespaces the control plane can access. Using a {k8s} label selector, the administrator sets the criteria for the namespaces visible to the control plane, excluding any namespaces that do not match the specified criteria.
+
```

Comment:
Avoid placing multiple notes next to each other. If multiple notes are necessary, restructure the information by moving the less-important statements into the flow of the main content. See https://redhat-documentation.github.io/supplementary-style-guide/#admonitions

Reply:
Moved the less important note below.